ArchiSteamFarm/Steam/Storage/BotConfig.cs

```diff
@@ -273,6 +276,9 @@ public sealed class BotConfig {
 	[JsonProperty(Required = Required.DisallowNull)]
 	public ArchiHandler.EUserInterfaceMode UserInterfaceMode { get; private set; } = DefaultUserInterfaceMode;
 
+	[JsonProperty(Required = Required.DisallowNull)]
+	public bool VRMode { get; private set; } = DefaultVRMode;
```

Comment:
Change this from `bool` to `SteamKit2.EPersonaStateFlag`. And name to something like `PersonaState`.

Reply:
Or name could also be something more user-friendly like "OnlineFlags" - it matches `OnlineStatus` we already have, it makes sense to me.